src/_reduceLazy.ts

```diff
@@ -1,6 +1,6 @@
-export type LazyResult<T> = LazyEmpty<T> | LazyNext<T> | LazyMany<T>;
+export type LazyResult<T> = LazyEmpty | LazyNext<T> | LazyMany<T>;
 
-interface LazyEmpty<T> {
+interface LazyEmpty {
```

Comment:
removed the generic because it's unused and it error'd when I imported `flatMap` in the docs building 😮